app/index.js

```diff
@@ -54,7 +54,8 @@ module.exports = generators.Base.extend({
         projectRoot: 'generators',
         skipInstall: this.options.skipInstall,
         readme: readmeTpl({
-          generatorName: this.props.name
+          generatorName: this.props.name,
+          generatorRealName: this.props.name.replace('generator-', '')
```

Comment:
What does `realName` mean? Can't we find a more clear name? Or simply do the replace call in the template file?